src/libraries/System.Windows.Extensions/src/System/Media/SoundPlayer.cs

```diff
@@ -310,7 +310,20 @@ private void LoadStream(bool loadSync)
                 int streamLen = (int)_stream.Length;
                 _currentPos = 0;
                 _streamData = new byte[streamLen];
-                _stream.Read(_streamData, 0, streamLen);
+#if NET7_0_OR_GREATER
+                _stream.ReadExactly(_streamData);
+#else
```

Comment:
Should we backport this?

Reply:
Same as https://github.com/dotnet/runtime/pull/82621#discussion_r1123485258